openmetadata-service/src/main/java/org/openmetadata/service/governance/workflows/elements/Edge.java

```diff
@@ -0,0 +1,30 @@
+package org.openmetadata.service.governance.workflows.elements;
+
+import org.flowable.bpmn.model.BpmnModel;
+import org.flowable.bpmn.model.Process;
+import org.flowable.bpmn.model.SequenceFlow;
+import org.openmetadata.common.utils.CommonUtil;
+
+public class Edge {
+  private final SequenceFlow edge;
+
+  public Edge(org.openmetadata.schema.governance.workflows.elements.EdgeDefinition edgeDefinition) {
+    SequenceFlow edge = new SequenceFlow(edgeDefinition.getFrom(), edgeDefinition.getTo());
+    if (!CommonUtil.nullOrEmpty(edgeDefinition.getCondition())) {
+      edge.setConditionExpression(getFlowableCondition(edgeDefinition.getCondition()));
+    }
+    this.edge = edge;
+  }
+
+  private String getFlowableCondition(boolean condition) {
+    if (condition) {
+      return "${result}";
+    } else {
+      return "${!result}";
+    }
```

Comment:
Why do we return `${result}` or `${!result}`; how is the condition used?

Reply:
It is due to how we define Flowable inner logic. For now it is fairly simple and we're just using Booleans. Eventually we need to sort out how to deal with Inputs/Outpus better.

In a nutshell, `result` is the variable we are using. This should be done through constants for sure!